src/utils/validators/graphqlQueryValidators.ts

```diff
@@ -98,18 +99,10 @@ export const createDonationQueryValidator = Joi.object({
   transactionNetworkId: Joi.string()
     .required()
     .valid(...Object.values(NETWORK_IDS)),
-  tokenAddress: Joi.when('transactionNetworkId', {
-    is: NETWORK_IDS.SOLANA_MAINNET,
+  tokenAddress: Joi.when('chainType', {
+    is: Joi.string().pattern(chainTypeRegex),
```

Comment:
Whoops I pushed wrong regex during testing. Fixed.

Reply:
So, why regex? :)
It only would match with "SOLANA" value which can be simple equal condition (then remove `chainTypeRegex` variable). Also, it's better to use ChainType.SOLANA instead of repeating "SOLANA" here.